src/design-system/organisms/MintBurnContainer/MintBurnContainer.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
```

Comment:
This component is now has been re-used 3rd time. `By/Sell`, `Liquidity` and `Mint/Burn`
Please make this implementation generic and re-use it.